### PR TITLE
update Brother integration examples with new template sensor format

### PR DESCRIPTION
**Pull request by cturra**
## Proposed change
template sensor configurations were changed in 2021.4 and the Brother printer integration documents still reference the legacy format.  just a quick update to correct that.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: N/A
- Link to parent pull request in the Brands repository: N/A
- This PR fixes or closes issue: N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @cturra, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19656).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/brother.markdown [hunk 1]
```diff
@@ -37,21 +37,23 @@ For some Brother devices, `SNMPv3 read-write access and v1/v2c read-only access`
...
+    - name: 'Laser Printer Out Of Paper'
+      unique_id: laser_printer_out_of_paper
+      state: >
+        {{ is_state('sensor.<DEVICE_NAME>_status', 'no paper') }}
+
+  - binary_sensor:
+    - name: 'Later Printer Paper Jam'
+      unique_id: laser_printer_paper_jam
+      state: >
+        {{ is_state('sensor.<DEVICE_NAME>_status', 'paper jam') }}
```
Comment (bieniu) [inline source/_integrations/brother.markdown]: I'm not sure if we should use `<DEVICE_NAME>` here, user may have changed entity_id. IMO it's better to use `sensor.hl_l2340d_status` as an example.

Comment (cturra) [reply source/_integrations/brother.markdown]: agreed.

File: source/_integrations/brother.markdown [hunk 2]
```diff
@@ -37,21 +37,23 @@ For some Brother devices, `SNMPv3 read-write access and v1/v2c read-only access`
...
+    - name: 'Laser Printer Out Of Paper'
+      unique_id: laser_printer_out_of_paper
+      state: >
+        {{ is_state('sensor.hl_l2340d_status', 'no paper') }}
+
+  - binary_sensor:
+    - name: 'Later Printer Paper Jam'
+      unique_id: laser_printer_paper_jam
+      state: >
+        {{ is_state('sensor.hl_l2340d_status', 'paper jam') }}
```
Comment (frenck) [inline source/_integrations/brother.markdown]: I would not recommend setting the `unique_id` in the example. For different reasons:

- It is optional.
- The suggested IDs I would not recommend (use of UUIDs is generally safer).

```suggestion
template:
  - binary_sensor:
    - name: 'Laser Printer Out Of Paper'
      state: >
        {{ is_state('sensor.hl_l2340d_status', 'no paper') }}

  - binary_sensor:
    - name: 'Later Printer Paper Jam'
      state: >
        {{ is_state('sensor.hl_l2340d_status', 'paper jam') }}
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @cturra 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**